apps/www/_blog/2023-07-06-postgrest-11.1-release.mdx

```diff
@@ -0,0 +1,218 @@
+  .from('big_table')
+  .select()
+  .eq('unindexed_column', 'value')
+```
+
+Then, immediately, the request will be aborted and the response will be:
+
+```jsx
+{
+  "hint": null,
+  "details": null,
+  "code": "54001",
+  "message": "plan cost limit exceeded"
+}
+```
+
+Note that tuning is required to get the cost limit right. You should use the `plan_filter.statement_cost_limit` with care as it can invalidate legitimate queries.
+
+## Configurable Transaction Isolation Level
+
+By default, all queries run in a transaction with the default *read committed* isolation level.
+
+You can now modify this with the `default_transaction_isolation` setting.
+
+If you want a function to run with *repeatable read* isolation level:
+
+```sql
+create function hello()
+returns text as $$
+  select 'hello';
+$$ language sql
+set default_transaction_isolation = 'repeatable read';
+```
+
+Or if you want an impersonated role to run its queries with a *serializable* isolation level:
+
+```sql
+alter role service_role set default_transaction_isolation = 'serializable';
+
+NOTIFY pgrst, 'reload config'; -- reload postgREST config cache
+```
+
+Note that the default *read committed* is good enough for almost all use cases. Higher isolation levels incur in overhead as they use more sophisticated locking. They're only needed in special cases.
+
+## Bulk insert JSON with default values
+
+A long wanted feature was bulk inserting JSON while considering columns' default values.
+
+Having the following sample table.
+
+```sql
+create table foo (
+  id bigint generated by default as identity primary key,
+  bar text,
+  baz int default 100
+);
+```
+
+You can now do it like this:
+
+```jsx
+const { error } = await supabase
+  .from('foo')
+  .insert([
+	  { "bar": "val1"
+	  }
+	, { "bar": "val2"
+	  , "baz": 15
+	  }
+	], defaultToNull: false)
+  .select()
+```
+
+And the response will be:
+
+```json
+[
+  { "id":  1
+  , "bar": "val1"
+  , "baz": 100
+  }
+, { "id":  2
+  , "bar": "val2"
+  , "baz": 15
+  }
+]
+```
+
+As you can see, `id` and `baz` took their default values.
+
+## ANY/ALL filter modifiers
+
+As a shortcut to `OR` filters, you can now use `any` modifiers on various filters. Take the `like` filter as an example:
+
+```jsx
+const res = await postgrest
+    .from('users')
+    .select()
+    .likeAnyOf('username', ['%supa%', '%kiwi%'])
+```
+
+This is equivalent to the following in SQL.
+
+```sql
+SELECT * FROM users WHERE username LIKE ANY('{%supa%,%kiwi%}');
+```
+
+`any` modifiers are available for the `eq,like,ilike,gt,gte,lt,lte,match,imatch` filters.
+
+For completeness, the `all` modifier is also included.
+
+## Closing up
+
+There you have it, now you can make your API more secure with role settings and use higher isolation levels without resorting to direct PostgreSQL connections.
+
+PostgREST v11.1 is available for all Supabase projects created after 5 July 2023. Existing projects will be gradually migrated over the next month.
```

Comment:
@kiwicopple In light of the recent [incident](https://status.supabase.com/incidents/jb5pzhjhbk0r), maybe we should put it like this:
```suggestion
PostgREST v11.1 is available for all Supabase projects created after 5 July 2023. Existing projects can upgrade by doing a pause/unpause.
```
WDYT?